xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/contentheader.vm

```diff
@@ -18,7 +18,8 @@
 ## 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 ## ---------------------------------------------------------------------------
 <div class="row document-header">
-  <div class="document-info col-xs-12 #if($displayContentMenu)col-md-7#end">
+  <section class="document-info col-xs-12 #if($displayContentMenu)col-md-7#end"
+    aria-labelledby="document-title">
```

Comment:
Is it really good to give that section the title of the page as label? The page title is not really a label of the section imho, something like "Page metadata" would be more fitting imho.

Reply:
Some info about the accessible name for region elements (default role of HTML of sections):

[region](https://w3c.github.io/aria/#region) Accessible name required 	
    Helps screen reader users understand the context and purpose of the landmark.
    Use aria-labelledby if a visible label is present, otherwise use aria-label.
    See the [Region Landmark](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/#aria_lh_region) section.

The title of the page is not enough :+1: 

I'm not sure I like the term 'metadata'. It's quite generic, and we have other sections that could use this name. Moreover, 'metadata' is a technical term and not friendly to non-technical users. 
I propose 'Page header', and 'Page actions' on the nav next to it.

https://trends.google.com/trends/explore?date=today%205-y&q=metadata,header&hl=en

Addressed in https://github.com/xwiki/xwiki-platform/pull/2133/commits/875e1746bcc7cc789df1a47831609ef8632c8ad8